manifest/3.1/3.1.2.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Copyright 2016-Present Couchbase, Inc.
+
+Use of this software is governed by the Business Source License included in
+the file licenses/BSL-Couchbase.txt.  As of the Change Date specified in that
+file, in accordance with the Business Source License, use of this software will
+be governed by the Apache License, Version 2.0, included in the file
+licenses/APL2.txt.
+-->
+
+<manifest>
+
+  <remote fetch="https://github.com/couchbase/" name="couchbase"/>
+  <default remote="couchbase" revision="master"/>
+
+  <!-- Build Scripts (required on CI servers) -->
+  <project name="product-texts" path="product-texts" remote="couchbase"/>
+  <project name="build" path="cbbuild" remote="couchbase" revision="5aaa2d5b2ccf13cb38ded87d8ef5c62b2ea65efc">
```

Comment:
This build commit looks like it's the SG commit - it should be the build commit used for the last 3.1.2 build.  From http://latestbuilds.service.couchbase.com/builds/latestbuilds/sync_gateway/3.1.2/34/sync_gateway-3.1.2-34-manifest.xml, I think this should be `844e2484067053d7af88e706a814766265bf03df`